lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -38,9 +38,24 @@
 from ansible.module_utils.connection import Connection, ConnectionError
 from ansible.module_utils.common._collections_compat import Mapping
 from ansible.module_utils.network.common.config import NetworkConfig, dumps
-from ansible.module_utils.six import iteritems, string_types
+from ansible.module_utils.six import iteritems, string_types, PY2, PY3
 from ansible.module_utils.urls import fetch_url
 
+try:
+    import yaml
+    HAS_YAML = True
+except ImportError:
+    HAS_YAML = False
+
+try:
+    if PY3:
+        from collections import OrderedDict
+    else:
+        from ordereddict import OrderedDict
```

Comment:
we should not be adding a python dependency on ordereddict here, as python2.7 can also use ordereddict from collections:

  https://docs.python.org/2/library/collections.html#collections.OrderedDict

This also means, that python2.7 users now need an additional python dependency installed.

Reply:
@pabelanger Thanks for the feedback. I'll tighten up the `ordereddict` import to pre-2.7.